bin/upgrade-wikis

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# This script can be used to run the APIs MediawikiUpdate Job for individual mediawiki databases
+# 
+# After this script is used you have to point the wikis to the new mediawiki backends.
+# Then you can remove the read-only mode again with the script `unlock-wikis`.
+#
+# `DOMAIN_FILE_LIST` is just the path to a plain-text file containing a list of wikibase.cloud domains, each line containing 1 wiki domain
+# in the form of:
+# ```
+# foo.wikibase.cloud 
+# bar.wikibase.cloud
+# ``
+
+set -exu
+
+# These need to be adjusted for each upgrade
+DOMAIN_FILE_LIST='./domains.txt'
+DB_VERSION_OLD='mw1.37-fp-wbs1'
```

Comment:
Suggestion: should these be passed as options, e.g.:

```
cat domains.txt | ./upgrade-wikis -f mw1.37-fp-wbs1 -t mw1.38-wbs1 -b mediawiki-138
```

Reply:
I'd love that but I always struggle with implementing something like this cleanly and readable in bash

Reply:
I guess `getopts` is the way to go (as seen in other scripts here as well), which coincidentally means we _do need_ bash to run this:

```
DB_VERSION_TO='mw1.37-fp-wbs1'
DB_VERSION_FROM='mw1.38-wbs1'
MW_BACKEND='mediawiki-138'

# :x means there is value associated to the flag, x only would be a boolean flag
while getopts ":f:t:b" option; do 
   case $option in
      f)
         DB_VERSION_FROM=$OPTARG;;
      t)
         DB_VERSION_TO=$OPTARG;;
      b)
         MW_BACKEND=$OPTARG;;
      # optionally add a default case that exits 1 on unknown flags?
   esac
done
```